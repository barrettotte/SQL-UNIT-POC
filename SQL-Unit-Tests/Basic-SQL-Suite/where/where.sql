-- This is a simple test for an WHERE statement --

SELECT * FROM [BARRETT_TEST].[dbo].[roles] WHERE role_name='ROLE_DEVELOPER';
