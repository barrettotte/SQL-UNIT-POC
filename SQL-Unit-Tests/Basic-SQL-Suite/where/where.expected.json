{
    "success": "true",
    "columns": [
        {
            "name": "id",
            "type": "BIGINT"
        },
        {
            "name": "role_name",
            "type": "VARCHAR(50)"
        }
    ],
    "rows": [
        {
            "role_name": "ROLE_DEVELOPER"
        }
    ]
}